format/spec.md

```diff
@@ -603,6 +608,10 @@ Notes:
 4. Position delete metadata can use `referenced_data_file` when all deletes tracked by the entry are in a single data file. Setting the referenced file is required for deletion vectors.
 5. The `content_offset` and `content_size_in_bytes` fields are used to reference a specific blob for direct access to a deletion vector. For deletion vectors, these values are required and must exactly match the `offset` and `length` stored in the Puffin footer for the deletion vector blob.
 6. The following field ids are reserved on `data_file`: 141.
+7. `geometry`, this is a point: X, Y, Z, and M take the min value of all component points of all geometries in file. See Appendix D for encoding.
+8. `geography`, this is a point: X = westernmost bound of all geometries in file, Y = northernmost bound of all geometries in file, Z is min value for all component points of all geometries in the file, M is min value of all component points of all geometries in the file. See Appendix D for encoding.
```

Comment:
This definition works in almost all cases, except when the longitudes span the entire space (think boundary of Antarctica). I am not sure this is important, and whether it needs to be specified further.

Reply:
I was further thinking about this: for the geographic case, we never formally specify that for the extremal values need to be in [-180,180] for longitudes and in [-90, 90] for latitudes. Under this restriction the only possible way to define the entire space of longitudes is to specify that the western most value is -180 and the easternmost 180.

Maybe it would be good to specify the allowed range for the extremal values for the geographic case.

Reply:
Thanks, this and next comment are now fixed.